lib/Service/Importer/BoardImportCommandService.php

```diff
@@ -179,21 +196,28 @@ public function bootstrap(): void {
 	public function import(): void {
 		$this->getOutput()->writeln('Starting import...');
 		$this->bootstrap();
-		$this->getOutput()->writeln('Importing board...');
-		$this->importBoard();
-		$this->getOutput()->writeln('Assign users to board...');
-		$this->importAcl();
-		$this->getOutput()->writeln('Importing labels...');
-		$this->importLabels();
-		$this->getOutput()->writeln('Importing stacks...');
-		$this->importStacks();
-		$this->getOutput()->writeln('Importing cards...');
-		$this->importCards();
-		$this->getOutput()->writeln('Assign cards to labels...');
-		$this->assignCardsToLabels();
-		$this->getOutput()->writeln('Importing comments...');
-		$this->importComments();
-		$this->getOutput()->writeln('Importing participants...');
-		$this->importCardAssignments();
+		$boards = $this->getImportSystem()->getBoards();
+
+		foreach ($boards as $board) {
+			$this->reset();
+			$this->setData($board);
+			$this->getOutput()->writeln('Importing board...');
```

Comment:
I think it would be nice to also inform the user which board we are working on when we start:
```suggestion
			$this->getOutput()->writeln('Importing board "' . $this->getBoard()->getTitle() . '".');
```
In particular so one knows which board caused problems if an exception kills the import or so.